detect_secrets/core/baseline.py

```diff
@@ -34,13 +34,15 @@ def initialize(
         exclude_lines=exclude_lines_regex,
     )
 
-    if os.path.isfile(path):
+    if len(path) > 1 or os.path.isfile(path[0]):
```

Comment:
I'm doubtful this carries over the same logic as the previous version.

After all, this should allow invocations like:

```
$ detect-secrets scan non-existent-file.A non-existent-file.B
```

right?

Reply:
Took a closer look at this and to be fair I think that I misinterpreted what the previous version was doing. So updated it to:
- Running `detect-secrets scan non-existent-file.A non-existent-file.B`  skips the invocation.
- Running `detect-secrets scan` or `detect-secrets scan detect-secrets` works as expected.
- Running `detect-secrets scan non-existent-file.A existent-file.B`  skips the invocation too! But I'm not sure if this is great or not. If one argument isn't a file scanning the other ones should work?

Do you think this is the proper way to solve this?

Reply:
That's a good question you raise. When it comes to command line tools usability, I defer back to tried-and-true unix tools:

```
detect-secrets $ cat asdf
cat: asdf: No such file or directory
detect-secrets $
detect-secrets $ cat asdf requirements-dev.txt
cat: asdf: No such file or directory
coverage
flake8==3.5.0
mock
monotonic
pre-commit==1.11.2
pytest
pyyaml
tox>=3.8
tox-pip-extensions
unidiff
```

Perhaps we should do something similar? You can print to stderr, or you can just use:

```
log.error(...)
```

Reply:
Added error logging!